packages/distcc/build.sh

```diff
@@ -0,0 +1,13 @@
+TERMUX_PKG_HOMEPAGE=http://distcc.org/
+TERMUX_PKG_DESCRIPTION="Distributed C/C++ compiler."
+TERMUX_PKG_VERSION=3.2-rc1
+TERMUX_PKG_REVISION=1
+TERMUX_PKG_SHA256=8cf474b9e20f5f3608888c6bff1b5f804a9dfc69ae9704e3d5bdc92f0487760a
+TERMUX_PKG_SRCURL=https://storage.googleapis.com/google-code-archive-downloads/v2/code.google.com/distcc/distcc-3.2rc1.tar.gz
```

Comment:
might want to consider morving to github's archives https://github.com/distcc/distcc/archive/v3.2rc1.tar.gz

Reply:
github archives suffer from https://github.com/distcc/distcc/issues/237

Reply:
what about adding this step to get around

```sh
termux_step_pre_configure() {
     ./autogen.sh
}
```